packages/libraries/cli/src/commands/schema/check.ts

```diff
@@ -60,7 +60,7 @@ const schemaCheckMutation = graphql(/* GraphQL */ `
           webUrl
         }
       }
-      ... on SchemaCheckError @skip(if: $usesGitHubApp) {
+      ... on SchemaCheckError {
```

Comment:
If the API returns `SchemaCheckError` (which it does when you use the `--github` flag without a `--service` flag on a federation project), we should still get these fields and print the error to the user.